src/bsp/platform/stm32fx_nucleo_wifi/xi_bsp_time_stm32f4_nucleo_wifi_sntp.c

```diff
@@ -6,6 +6,11 @@
 #include "wifi_interface.h"
 #include "xi_bsp_time_stm32f4_nucleo_wifi_sntp.h"
 
+#define SNTP_PORT   123
+#define SNTP_SERVER_HOST_LEN 15
+char* sntp_servers[] = {"0.pool.ntp.org", "1.pool.ntp.org",
```

Comment:
in xi_bsp_time_cc3200_sntp.c Bob uses these servers
"pool.ntp.org", "time-a.nist.gov", "time-b.nist.gov", "time-c.nist.gov"
These four sound more ... hm ... reliable, or might have higher aggregated uptime compared to *.pool.ntp.org.

Just guessing, so if you don't think I am ok with your selection.